src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -200,6 +200,11 @@ static void Main(string[] args)
             //Block until all Sinks have been terminated.
             TestRunSystem.WhenTerminated.Wait(TimeSpan.FromMinutes(1));
 
+            if (Debugger.IsAttached)
+            {
+                Console.ReadLine(); //block when debugging
```

Comment:
Added this to make it easier to debug multi-node tests when launching them directly from visual studio.